birdhouse/optional-components/stac-data-proxy/config/magpie/permissions.cfg.template

```diff
@@ -0,0 +1,12 @@
+# NOTE:
+# Assume 'secure-data-proxy' would exist if needed.
```

Comment:
This line is confusing. Do you mean: "The permissions in this file only take effect if the secure-data-proxy component is enabled" ?